README.md

```diff
@@ -447,6 +447,14 @@ For more information, see our [module contribution guide.](https://docs.puppet.c
 * The Puppet types are built from the source code of each DSC Resources MOF schema files. If you want to build the types, read the [Quick-start and Building](README_BUILD.md#quick-start).
 * If you want the build Puppet types for your own custom DSC Resources, read [Build Custom DSC Resource Types](README_BUILD.md#build-custom-dsc-resource-types).
 
+### Version Strategy
+
+This module generally follows [Semantic Versioning](http://semver.org/) for choosing an appropriate release version number with the following exception;
+
+* Minor e.g. from version 2.0.0 to 2.1.0
+
+A minor change may include [rebuilding the DSC resource types](README_BUILD.md).  Puppet wants to keep pace with the released DSC Resources from the PowerShell team repository, but this engenders risk as Puppet adopts third party code. Normally this would mean making major version bumps, but since this is anticipated to be frequent that would be too much churn. Instead the minor version will indicate when types are regenerated.
```

Comment:
* I agree with Moses, we try to avoid "will".
* There's a double space after the first sentence that needs removing.
* Lowercase the R in `DSC Resources`.

Thanks!